README.md

```diff
@@ -39,6 +39,16 @@ $ docker build -t blackducksoftware/bdio-tinkerpop-db bdio-tinkerpop-db/
 $ docker run -d -e POSTGRES_HOST_AUTH_METHOD=trust -p 5432:5432 blackducksoftware/bdio-tinkerpop-db
 $ ./gradlew build
 ````
+## Release:
+__As of 09-28-23:__
+To release a version of bdio to artifactory for use by downstream dependencies:
```

Comment:
From my last experience, which happened on 11-03-2023, you actually should add -SNAPSHOT (if it is not already there). So, if you want to release 3.2.10, gradle properties should have 3.2.10-SNAPSHOT